ledgereth/__main__.py

```diff
@@ -86,9 +86,24 @@ def get_args(argv):
         "-p",
         "--gasprice",
         type=int,
-        required=True,
+        required=False,
         help="The gas price to use for the tx",
     )
+    send_parser.add_argument(
+        "-f",
+        "--max-fee",
+        type=int,
+        required=False,
+        help="The max fee per gas to use for the tx",
+    )
+    send_parser.add_argument(
+        "-b",
+        "--bribe",
```

Comment:
This is the priority fee, right?
I wonder if aliasing this to `bribe` may confuse some users

Reply:
Yeah, the words "priority fee" and "base fee" are typically the chosen words for describing EIP1559

Reply:
Original EIP draft used the word "bribe" (*which is what it is*), but this is pretty opinionated and probably just adds to confusion.  I'll change it.